README.md

```diff
@@ -47,19 +60,42 @@ val v1CartCreatedPersister = persister[CartCreatedV1]("cart-created")
 
 // spray-json persister for V2 but with support for migration
 // of data writen in the V1 format.
-val v2CartCreatedPersister = persister[CartCreatedV2, V2]("cart-created",
-  from[V1].to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
+val v2CartCreatedPersister = persister[CartCreatedV2, V2](
+  "cart-created",
+  from[V1]
+    .to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
 )
 
 // spray-json persister for V3 but with support for migration
 // of data writen in the V1 and V2 formats.
-val v3CartCreatedPersister = persister[CartCreatedV3, V3]("cart-created",
+val v3CartCreatedPersister = persister[CartCreatedV3, V3](
+  "cart-created",
   from[V1]
     .to[V2](_.update('cart / 'items / * / 'price ! set[Int](1000)))
     .to[V3](_.update('timestamp ! set[Long](System.currentTimeMillis)))
 )
 ```
 
+To use persisters, they need to be encapsulated in a `StaminaAkkaSerializer` that is registered, in the 
+application.conf, for the event classes that need to be persisted. Here the provided `Persistable` marker trait is used
```

Comment:
There's a few too many commas in that sentence :smile: I also think it might need some more context.

How about:

> For these persisters to be actually used by the Akka serialization system, you will need to bundle them into an Akka serializer and then register that serializer for all you classes. To make that registration process a little simpler Stamina comes with a very simple marker trait called `Persistable` and of course you can use your own marker traits instead.
>
> In the below example we create a subclass of `StaminaAkkaSerializer` and pass all our `Persister` instances into it. We then register this serializer with Akka in our application.conf and bind it to all instances/subclasses of our `Persistable` marker trait.